editor/src/core/tailwind/tailwind-compilation.ts

```diff
@@ -16,6 +16,30 @@ import type { RequireFn } from '../shared/npm-dependency-types'
 import { TailwindConfigPath } from './tailwind-config'
 import { ElementsToRerenderGLOBAL } from '../../components/canvas/ui-jsx-canvas'
 import { isFeatureEnabled } from '../../utils/feature-switches'
+import type { Config } from 'tailwindcss/types/config'
+import type { EditorState } from '../../components/editor/store/editor-state'
+import { createRequireFn } from '../property-controls/property-controls-local'
+
+const LatestConfig: { current: { code: string; config: Config } | null } = { current: null }
+export function getTailwindConfig(editorState: EditorState): Config | null {
+  const tailwindConfig = getProjectFileByFilePath(editorState.projectContents, TailwindConfigPath)
+  if (tailwindConfig == null || tailwindConfig.type !== 'TEXT_FILE') {
+    return null
+  }
+  const cached =
+    LatestConfig.current == null || LatestConfig.current.code !== tailwindConfig.fileContents.code
+      ? null
+      : LatestConfig.current.config
+
+  if (cached != null) {
+    return cached
+  }
+  const { customRequire } = createRequireFn(editorState, TailwindConfigPath)
```

Comment:
let's add a FIXME here that this should use a shared long-lived require function instead of creating a brand new one

Reply:
https://github.com/concrete-utopia/utopia/pull/6500/commits/0b2a8404813c53b004a1dbb2566c627c9e3ed596